app/konnect/dev-portal/applications/application-overview.md

```diff
@@ -20,7 +20,37 @@ You can perform the following actions from an application's details page:
 - View the status of an application registration to an API product.
 - Open the analytics dashboard and view metrics about an application.
 
-The guides within this section cover these topics:
+## Application authentication considerations
+
+### Support for any control plane
+
+App registration is fully supported in the `default` control plane when using the application `consumers` and the `acl` plugin. The `default` control plane is the one that is first created in each geo when you create an organization.
+For non-`default` control planes, app registration is supported using the `konnect-application-auth` plugin available as of {{site.base_gateway}} 3.0. <!-- We mention using the different plugins, what do we need to do in those plugins to support app registration? Why do default CP and non-default CPs use different plugins and why those particular plugins? -->
```

Comment:
This is indeed a complicated part or the product to explain. Here is some context and explanation:

Initially app registration in Konnect was implemented using a combination of `acl`, `key-auth`, and `openid-connect` plugins which are included in every version of Kong Gateway that is supported by Konnect. Using these plugins gives the benefits that they are compatible with Gateway `consumers` which can be mapped to portal applications, providing further plugin functionality like applying `rate-limiting` plugins based on the application `consumer`, etc.

This feature was developed when Konnect had only a single Control Plane per organization.

However, at the time that Konnect rolled out support for multiple Control Planes, the application registration feature was re-imagined so that it could scale to support APIs that might live in different control planes. As part of this, the plugins, keys, and consumers previously used were dropped from the design and replaced with a new plugin `konnect-application-auth`, credential management system, and access control system.

Migrating existing customers who were using the "legacy" (consumer-based) app registration design was not fully possible, however, because:
- Some customers were relying on the use of application `consumers` which are no longer supported
- The new design (and `konnect-application-auth` plugin) is only available on Kong Gateway versions 3.0+, and some customers were unable to upgrade from a 2.x version

This left the system in the following state:
- the (first) control plane in all existing orgs became known as the `default` control plane which would continue to support certain legacy behaviors (including consumer-based app reg) that would not be supported in newly created control planes
- new orgs would still have their first control plane be considered `default` and use the legacy consumer-based app registration
- additional control planes created in any org would be considered "non-default"/non-legacy and use the new design centered around the `konnect-application-auth` plugin. These control planes are not compatible with Kong Gateway versions < 3.0

This has been the case up until recently, where an update was made so that new orgs will no longer receive a control plane that is considered "default" or legacy, and there is no longer support for using Kong Gateway < 3.0 with app registration in new orgs. The exception is "legacy mode" for app registration can still be turned on for any control plane if requested via support.

I hope that helps to clarify some of the context and I would be happy to discuss this more if needed.

Reply:
@Mierenga Thanks for the info! This is really helpful. Just to make sure I understand, the content I currently had would only apply to legacy users (whether they got Konnect before 3.0 or because they requested legacy through support), this wouldn't apply to new orgs? I wanted to delete this section about control planes, but I figured I should keep it with a note about "If you started using Konnect pre 3.0 gateway, keep the following in mind:" or something like that. What do you think?

Reply:
Thanks! I'll make those edits